doc/Extensions/RRDCached.md

```diff
@@ -5,10 +5,6 @@ path: blob/master/doc/
 
 This document will explain how to setup RRDCached for LibreNMS.
 
-Since version 1.5, rrdtool / rrdcached now supports creating rrd files
-over rrdcached. If you have rrdcached 1.5.5 or above, you can also
-tune over rrdcached. To enable this set the following config:
-
 ```php
 $config['rrdtool_version'] = '1.5.5';
 ```
```

Comment:
Should we keep the config and the note ?

Reply:
i think it could be savely removed.

even Debian Stable (Buster) has RRDTool 1.7.1 and RRDCached 1.7.1

rrdtool 1.5.5. was released [Nov 2015](https://github.com/oetiker/rrdtool-1.x/releases)

Reply:
CentOS 7 comes with 1.4.8

Reply:
mmh, as i see here it's very old, but still fully maintained.
[https://en.wikipedia.org/wiki/CentOS#Versioning_and_releases](https://en.wikipedia.org/wiki/CentOS#Versioning_and_releases)

Ok i say it should stay in it.
Maybe with a hint since which Distributions Release names/versions tune over rrdcached is possible.

Reply:
All right, so we need to keep this, or even better, we need to figure out the version of rrdtool (could be done in daily for instance) and set this automatically. That way, we would avoid the manual setup.